### PR TITLE
WPC-728 tmp user bug

### PR DESCRIPTION
**Pull request by michael-scheurer**

### CODE REVIEW

**Review by roosnic1 — COMMENTED** [review]
@tomaszdurka looks good to me. But why do you change an existing migration? AFAIK this migration has already run on all operators. I think it makes more sense to put those changes in a new migration so it runs for everybody.

**Comment by elias-summermatter** [comment]
@Itrulia No its not this pull request is here to migrate all tempuser securly

File: yarn.lock [hunk 1]
```diff
@@ -5554,50 +5554,6 @@
...
-  version "2.15.0"
-  resolved "https://registry.npmjs.org/@wepublish/api-db-mongodb/-/api-db-mongodb-2.15.0.tgz#441f6bc6be6f27c70ba42555a5ed0cb8b5de92c1"
-  integrity sha512-FzcpRfu6QTi1HkpHF7/km71fxAdVs8FazDo6m3XFXPsCUMRfHM/YxSKtUAZQI7EafHMczPxHWn8J9C5yFj8h5g==
-  dependencies:
-    bcrypt "^5.0.0"
-    mongodb "^3.5.4"
-    nanoid "^2.1.11"
-
-"@wepublish/api-media-karma@*":
-  version "2.15.0"
```
Comment (elias-summermatter) [inline yarn.lock:5567]: Why do you remove all this packages?

**Review by elias-summermatter — CHANGES_REQUESTED** [review]

**Comment by tomaszdurka** [comment]
@michael-scheurer any update here for @elias-summermatter comment?

File: yarn.lock (hunk 1 above)
Comment (michael-scheurer) [reply yarn.lock]: thanks for the review. a mistake it was.

**Review by tomaszdurka — APPROVED** [review]

**Pull request merged**